code/modules/antagonists/eldritch_cult/eldritch_book.dm

```diff
@@ -20,7 +18,6 @@
 	. = ..()
 	if(!IS_HERETIC(user))
 		return
-	. += "The Tome holds [charge] charges."
 	. += "Use it on the floor to create a transmutation rune, used to perform rituals."
```

Comment:
```suggestion
```

#3450

Reply:
UPDATE: This will be addressed via a separate PR: #3757

Reply:
```suggestion
	. += "The Tome holds [charge] charges."
```